src/shared_modules/utils/reflectiveJson.hpp

```diff
@@ -158,12 +158,12 @@ inline bool isEmpty([[maybe_unused]] bool value)
 
 inline bool isEmpty(std::string_view value)
 {
-    return value.empty();
+    return value.empty() || value.find_first_not_of(" \t\n\r\f\v");
```

Comment:
- I'd compare the result of `find_first_not_of` against `npos` instead of using it directly in the OR expression
- We are discarding any string that contains a space and we shouldn't 
- I'm not sure about the special characters

Reply:
Changed!